src/test/java/uk/gov/companieshouse/filinghistory/api/mapper/upsert/AbstractTransactionMapperFactoryTest.java

```diff
@@ -64,16 +64,15 @@ void shouldReturnAssociatedFilingTransactionMapperWhenAnnotationKindPassed() {
         assertEquals(associatedFilingTransactionMapper, actualMapper);
     }
 
-    // REMOVE TEST WHEN RESOLUTIONS ARE IMPLEMENTED
     @Test
-    void shouldReturnInvalidMapperExceptionWhenKindDoesNotReturnAMapper() {
```

Comment:
Keep this test, change exception

Reply:
will break if not all enum cases are covered